config/initializers/assets.rb

```diff
@@ -3,9 +3,12 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 
-# Add additional assets to the asset load path
+# Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
+# Add Yarn node_modules folder to the asset load path.
+Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
I'm not sure if this change is ok, happy to revert (or just comment it out) if this is wrong.

Reply:
I think we should remove this (or comment it out) unless it's somehow necessary to get the Admin UI working -- are you able to boot up the app locally and get into the admin interface to test it?

Reply:
Yup turns out this isn't needed, so I commented this out; just waiting on specs now